Cup_of_join.py

```diff
@@ -1,16 +1,22 @@
 def join(*args, sep='-'):
-    if args is None:
+    """
+    This function get an unlimited number of lists and a character.
+    The function return a new list consisting of the contents of all the given lists.
+    The contents of all the lists in the new list seperated by the given character.
+    """
```

Comment:
documentation does not adhere to the documentation conventions. no separate explanation of the parameters and of what the function returns. Please rewrite the documentation according to the conventions.